src/lib/util/resolveWithStatus.server.ts

```diff
@@ -10,7 +10,7 @@ import type { MaybePromise, RequestEvent } from "@sveltejs/kit";
 const resolveWithStatus = async <T extends RequestEvent>(
   status: number,
   statusText: string,
-  resolve: (event: T) => MaybePromise<Response>,
```

Comment:
`MaybePromise` is no longer exported, so I took the type and manually entered it here.